src/table/cells/handle-key-press.js

```diff
@@ -50,21 +54,37 @@ export const preventDefaultBehavior = (evt) => {
   evt.preventDefault();
 };
 
-const handleKeyPress = (evt, rootElement, rowIndex, colIndex) => {
-  preventDefaultBehavior(evt);
+const handleKeyPress = (evt, rootElement, rowIndex, colIndex, cell, selState, selDispatch) => {
   switch (evt.key) {
-    // TODO page up/down etc
     case 'ArrowUp':
     case 'ArrowDown':
     case 'ArrowRight':
     case 'ArrowLeft': {
+      preventDefaultBehavior(evt);
       removeCurrentFocus(evt);
       const rowAndColumnCount = getRowAndColumnCount(rootElement);
-      const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex);
+      const { nextRow, nextCol } = arrowKeysNavigation(evt, rowAndColumnCount, rowIndex, colIndex, selState);
       moveToNextFocus(rowAndColumnCount.rowElements, nextRow, nextCol);
       break;
     }
-    // TODO handle selections, search?, sorting...
+    // Space bar: Selects value.
+    case ' ': {
+      preventDefaultBehavior(evt);
+      cell?.isDim && selectCell(cell, selState, selDispatch, evt);
```

Comment:
Do we evet check if selections are enabled? `selState.isEnabled`

Reply:
no, we don't. 
I do not think that it is needed here. As when selection is not enabled, it is impossible to get focus on a cell and use the space bar to do the selection.

Reply:
Just tried it, you still can. In edit mode for example. But that is because of the hacky solution that the first cell has tabindex 0. As soon as you have focus on a cell, you can do whatever you want. I think we will start with only being able to tab into the chart in a proper way (working on it. now), and have all the functionality available when navigating. So this is fine for now

Reply:
I agree. Thanks.